cf_units/_udunits2_parser/README.md

```diff
@@ -24,9 +25,20 @@ changes to the grammar being proposed so that the two can remain in synch.
 
 ### Updating the ANTLR version
 
-The above script downloads a Java Jar which needs updating to the same version
-as antlr4-python3-runtime specified in the python requirements.  Once these have
-been updated, run [the script](compile.py) to regenerate the parser.
+The [compile.py script](compile.py) copies the ANTLR4 runtime into the _antlr4_runtime
+directory, and this should be commited to the repository. This means that we do not
+have a runtime dependency on ANTLR4 (which was found to be challenging due to the
+fact that you need to pin to a specific version of the ANTLR4 runtime, and aligning
+this version with other libraries which also have an ANTLR4 dependency is impractical).
+
+Given that the ANTRL4 runtime is vendored into this project, we have little need
+to upgrade the version unless new features of the parser/lexer generators are needed.
```

Comment:
```suggestion
Since the generated code is committed to this repo, and the ANTRL4 runtime is also vendored into it, we won't ever need to run ANTLR4 unless the grammar changes.

So, we will  only change the ANTLR4 version if we need new features of the
parser/lexer generators, or it becomes difficult to support the older version.

```